src/test/java/PlaylistTests.java

```diff
@@ -0,0 +1,60 @@
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.LoginPage;
+
+public class PlaylistTests extends BaseTest{
+
+    @Test
+    public void deletePlaylist() throws InterruptedException {
+
+        LoginPage loginPage = new LoginPage(getThreadLocal());
+        HomePage homePage = new HomePage(getThreadLocal());
+        loginPage.login();
+        homePage.selectPlaylist();
+        homePage.clickDeletePlaylistBtn();
+        Thread.sleep(2000);
+        homePage.getPlaylistDeletedMessage();
+
+       // Assert.assertEquals(getPlaylistDeletedMessage(),deletedPlaylistSuccessMessage);
+        Assert.assertTrue(homePage.getPlaylistDeletedMessage());
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.